### PR TITLE
[ICON SCREEN] [SWE-1630] Remake Icon Screen

### PR DESCRIPTION
**Pull request by AnastasiaShapowalova**
Screenshot 2022-07-27 at 21 25 09

### CODE REVIEW

**Comment by AnastasiaShapowalova** [comment]
- rewrote IconographyExamplesScreen
- transferred Olena's offset changes
- added comments and propTypes

File: src/app/domains/Iconography/components/IconList/IconList.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { ICONS_LIST } from '~/app/constants'
+import { IconListStyled } from './IconList.styled'
+import IconView from '~/app/domains/Iconography/components/IconView'
+
+const IconList = () => {
+  return (
+    <IconListStyled>
+      {ICONS_LIST?.map((item) => (
```
Comment (alexpassss) [inline src/app/domains/Iconography/components/IconList/IconList.js:8]: Useless condition ?.

File: src/app/components/bundles/ContentScreen/components/ContentScreenContent/ContentScreenContent.js [hunk 2]
```diff
@@ -1,20 +1,44 @@
...
+      ? DEFAULT_SCREEN_CONTENT_COLUMN_WIDTH - offset
+      : DEFAULT_SCREEN_CONTENT_COLUMN_WIDTH
 
   return (
     <>
-      <Col cw={1} />
-      <Col v="center" cw={7}>
-        {content}
-      </Col>
+      {offset !== 0 && <Col cw={offset} />}
```
Comment (alexpassss) [inline src/app/components/bundles/ContentScreen/components/ContentScreenContent/ContentScreenContent.js:29]: Wrong usage of Col

1. Bootstrap has offset classes out of the box
2. Col shouldn't "live" without Row

Read comments for Artem's latest PR

File: src/app/domains/Iconography/components/IconList/IconList.styled.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+const IconListStyled = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  gap: var(--icon-list-gap);
+  justify-content: center;
+  margin-top: auto;
+`
+
+export { IconListStyled }
```
Comment (alexpassss) [inline src/app/domains/Iconography/components/IconList/IconList.styled.js:11]: Wrong approach for exporting

File: src/app/domains/Iconography/components/IconView/IconView.styled.js [hunk 4]
```diff
@@ -4,7 +4,7 @@ const IconViewStyled = styled.div`
   display: flex;
   flex-direction: column;
   align-items: center;
-  width: var(--brandbook-module-icon-view-width);
+  width: var(--icon-view-width);
 `
 
 export { IconViewStyled }
```
Comment (alexpassss) [inline src/app/domains/Iconography/components/IconView/IconView.styled.js:10]: No reason to put it into the object. Use export default instead

File: src/app/components/bundles/ContentScreen/ContentScreen.js [hunk 5]
```diff
@@ -2,12 +2,27 @@ import {
   ContentScreenContainer,
   ContentScreenContent,
   ContentScreenInfo
-} from 'components/bundles/ContentScreen/components'
+} from '~/app/components/bundles/ContentScreen/components'
```
Comment (alexpassss) [inline src/app/components/bundles/ContentScreen/ContentScreen.js:5]: Strange path. Should be like that

Comment (Olexandr25) [reply src/app/components/bundles/ContentScreen/ContentScreen.js:5]: I took this path from fluqs, for fast moving inside component we write "~/"

File: src/app/domains/Iconography/components/IconView/IconView.js [hunk 6]
```diff
@@ -1,22 +1,34 @@
...
+  const { name } = props
 
   return (
     <IconViewStyled>
-      <Box mb={3}>{src && <Icon src={src} alt={name} />}</Box>
-      {name && <Text color="dark">{name}</Text>}
+      {name && (
+        <>
+          <Box mb={3}>
+            <Icon icon={name} size="72px" />
```
Comment (alexpassss) [inline src/app/domains/Iconography/components/IconView/IconView.js:21]: Hard coded value in pixels

File: src/styles/vars/theme.light.css [hunk 7]
```diff
@@ -120,17 +120,17 @@
...
   /* BORDER */
-  --bс-accent: var(--accent-default);
+  --bc-accent: var(--accent-default);
 
-  --bс-primary: var(--neutral-default);
-  --bс-primary-lighten3: var(--neutral-t-lighten3);
-  --bс-primary-lighten4: var(--neutral-t-lighten4);
-  --bс-primary-lighten5: var(--neutral-t-lighten5);
+  --bc-primary: var(--neutral-default);
+  --bc-primary-lighten3: var(--neutral-t-lighten3);
```
Comment (alexpassss) [inline src/styles/vars/theme.light.css:126]: Possible to improve naming
Not sure that --bc-primary-lighten3 is the best in such case

Comment (AnastasiaShapowalova) [reply src/styles/vars/theme.light.css]: I hope that I changed it in correct way

File: src/styles/vars/theme.light.css [hunk 8]
```diff
@@ -102,56 +102,34 @@
...
   /* BORDER */
-  --bс-accent: var(--accent-default);
+  --bc-accent: var(--accent-default);
 
-  --bс-primary: var(--neutral-default);
-  --bс-primary-lighten3: var(--neutral-t-lighten3);
-  --bс-primary-lighten4: var(--neutral-t-lighten4);
-  --bс-primary-lighten5: var(--neutral-t-lighten5);
+  --bc-primary: var(--neutral-default);
+  --bc-primary-l3: var(--neutral-t-lighten3);
```
Comment (alexpassss) [inline src/styles/vars/theme.light.css:125]: Still bad naming

It should have abstract meaning

File: src/app/domains/Iconography/components/IconView/IconView.js [hunk 9]
```diff
@@ -1,22 +1,34 @@
...
+  const { name } = props
 
   return (
     <IconViewStyled>
-      <Box mb={3}>{src && <Icon src={src} alt={name} />}</Box>
-      {name && <Text color="dark">{name}</Text>}
+      {name && (
+        <>
+          <Box mb={3}>
+            <Icon icon={name} size="4.5rem" />
```
Comment (alexpassss) [inline src/app/domains/Iconography/components/IconView/IconView.js:21]: Shouldn't be hard coded size

File: src/app/components/elements/Media/Icon/Icon.js [hunk 10]
```diff
@@ -1,4 +1,4 @@
-import { ICONS_NAMES, ICONS_VARIANTS } from '~/app/constants'
+import { ICONS_NAMES, ICONS_SIZES, ICONS_VARIANTS } from '~/app/constants'
```
Comment (alexpassss) [inline src/app/components/elements/Media/Icon/Icon.js:1]: strange path with ~

**Review by alexpassss — APPROVED** [review]

**Pull request merged**